data-prepper-pipeline-parser/src/main/java/org/opensearch/dataprepper/pipeline/parser/rule/RuleEvaluator.java

```diff
@@ -54,18 +54,22 @@ public RuleEvaluatorResult isTransformationNeeded(PipelinesDataFlowModel pipelin
     private RuleEvaluatorResult isDocDBSource(PipelinesDataFlowModel pipelinesModel) {
         PLUGIN_NAME = "documentdb";
         String pluginRulesPath = transformersFactory.getPluginRuleFileLocation(PLUGIN_NAME);
+        File ruleFile = new File(pluginRulesPath);
+        LOG.info("Checking rule path {}",ruleFile.getAbsolutePath());
         Map<String, PipelineModel> pipelines = pipelinesModel.getPipelines();
 
         for (Map.Entry<String, PipelineModel> entry : pipelines.entrySet()) {
             try {
                 String pipelineJson = OBJECT_MAPPER.writeValueAsString(entry);
                 if (evaluate(pipelineJson, pluginRulesPath)) {
-                    LOG.debug("Rule path {} is evaluated true for pipelineJson {}",pluginRulesPath, pipelineJson);
+                    LOG.info("Rule path {} is evaluated true for pipelineJson {}",pluginRulesPath, pipelineJson);
 
-                    String templateFilePath = transformersFactory.getPluginTemplateFileLocation(PLUGIN_NAME);
-                    PipelineTemplateModel templateModel = yamlMapper.readValue(new File(templateFilePath),
+                    String templateFilePathString = transformersFactory.getPluginTemplateFileLocation(PLUGIN_NAME);
+                    File templateFile = new File(templateFilePathString);
```

Comment:
Another way to access the file in resources directory is to use `getClass().getClassLoader().getResourceAsStream()` where you can use relative path.